### PR TITLE
MEGAcmd: Add version 1.1.0

### PR DESCRIPTION
**Pull request by issaclin32**
close lukesampson/scoop-extras#2597

**MEGAcmd** ([homepage](https://mega.nz/cmd)) is a command line tool to access MEGA(mega.nz) maintained by mega.nz itself.

Notes:
* **installer/uninstaller script**: 
MEGAcmd contains about 50 batch files for commands like `mega-get`, `mega-ls`, ...
These commands can also be executed by `MEGAclient.exe get`, `MEGAclient.exe ls`, and so on.

~There are 3 possible ways (I can think of) to handle this:~
~1. Ignore these commands and add `MEGAclient.exe` only.~
~2. Map all the available commands into `shim` directory.~
~3. use `env_add_path` (adding possibility of conflict with other programs on DLL files)
I think `2.` is the best approach.~

(put every batch file in `bin` instead)

### CODE REVIEW

File: bucket/megacmd.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "homepage": "https://mega.nz/cmd",
+    "description": "Command line tool to access MEGA(mega.nz)",
```
Comment (Ash258) [inline bucket/megacmd.json]:
```suggestion
    "description": "Manage MEGA.nz account and files from command line.",
```

File: bucket/megacmd.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://github.com/meganz/MEGAcmd/blob/master/build/installer/terms.txt"
+    },
+    "url": "https://mega.nz/MEGAcmdSetup.exe#/dl.7z",
+    "hash": "9a7ddc65cc6a792af2d5f54d69b2e08a5a48db050c8d3131011805b508a2f20d",
+    "installer": {
+        "script": [
+            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst.exe\" -Recurse",
+            "Get-ChildItem \"$dir\" 'mega-*.bat' | ForEach-Object {",
```
Comment (Ash258) [inline bucket/megacmd.json]: Just add them all as bin. What's problem?

Comment (Ash258) [reply bucket/megacmd.json]: Doing this manually is nonsense and dangerous.

Comment (Ash258) [reply bucket/megacmd.json]: `gci . 'mega-*.bat' | Select -expand Basename | % { "\"$_\"," } | clip` and paste it into manifest

Comment (issaclin32) [reply bucket/megacmd.json]: There may be more batch file added in the future. 
If I simply put every thing in `bin`, new commands will be ignored. 
(Until some user found out and file an issue)

Comment (Ash258) [reply bucket/megacmd.json]: And? That is casual.

Comment (issaclin32) [reply bucket/megacmd.json]: You mean just add `gci...` to the install script? Or replacing the whole part after `Get-ChildItem`?

Comment (Ash258) [reply bucket/megacmd.json]: No ...

1. Open manifest inside text editor
1. Create bin property as array.
1. Execute `gci . 'mega-*.bat' | Select -expand Basename | % { "\"$_\"," } | clip` inside terminal
1. Press CTRL+V in bin property inside manifest
1. Remove trailing comma
1. Press shortcut for formatting json file
1. Commit changes

Comment (issaclin32) [reply bucket/megacmd.json]: Thanks for the instruction.
I found that batch files in `bin` will not work without adding `.bat` in the end.

File: bucket/megacmd.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    "homepage": "https://mega.nz/cmd",
+    "description": "Manage MEGA.nz account and files from command line.",
+    "version": "1.1.0",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://github.com/meganz/MEGAcmd/blob/master/build/installer/terms.txt"
+    },
+    "url": "https://mega.nz/MEGAcmdSetup.exe#/dl.7z",
+    "hash": "9a7ddc65cc6a792af2d5f54d69b2e08a5a48db050c8d3131011805b508a2f20d",
+    "bin": [
```
Comment (Ash258) [inline bucket/megacmd.json]:
```suggestion
    "bin": [
        "MEGAclient.exe",
```

**Pull request merged**